README.adoc

```diff
@@ -29,13 +29,27 @@ buildPlugin()
   execute the steps against in parallel
 * `jenkinsVersions`: (default: `[null]`) - a matrix of Jenkins baseline versions to build/test against in parallel (null means default,
   only available for Maven projects)
+* `findbugs`: (default: `null`) - a map of parameters to run findbugs and/or archive findbugs reports. (only available for Maven projects)
+** `run`: set to `true` to add the `findbugs:findbugs` goal to the maven command.
+** `archive`: set to `true` to collect the findbugs report.
+** `pattern`: (default: `'**/findbugsXml.xml'`) the file pattern for the report.
```

Comment:
Why is this even customizable? Maven always puts the results in a predictable place.